src/components/DeliverableAnalysisDetailPage/DeliverableAnalysisDetailPage.tsx

```diff
@@ -54,6 +67,12 @@ export const DeliverableAnalysisDetailPage = () => {
 
   const deliverableAnalysis = serviceContainerDeliverableAnalysisOperation.data;
 
+  useEffect(() => {
```

Comment:
This `useEffect` (including its content) should be avoided because it's causing unnecessary request that needs to be canceled.